src/main/java/bio/terra/janitor/app/configuration/IamConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.janitor.app.configuration;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.type.TypeFactory;
+import java.io.IOException;
+import java.util.Set;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@Component
+@EnableConfigurationProperties
+@EnableTransactionManagement
+@ConfigurationProperties(prefix = "iam")
+public class IamConfiguration {
+  private static final Logger logger = LoggerFactory.getLogger(IamConfiguration.class);
+
+  // The file path for admin user list.
+  private String adminUserList;
+  private Set<String> adminUsers;
+  private boolean configBasedAuthZEnabled;
```

Comment:
Added some comments. Thanks.
And I would expect this property to long-lived because we want to disable the authZ check for local server(I changed the value in `run_local.sh`).
And by having this script, it would help for future migration to SAM?